#!/bin/sh /usr/share/dpatch/dpatch-run
## 03-locale.dpatch by Michael Bunk <azeem@debian.org>
##
## DP: Adjusts path for locale directory (Closes: #401674).

@DPATCH@

diff -Naur tinyerp-client-4.0.0-rc1.orig/bin/tinyerp-client.py tinyerp-client-4.0.0-rc1/bin/tinyerp-client.py
--- tinyerp-client-4.0.0-rc1.orig/bin/tinyerp-client.py	2006-11-27 12:30:44.000000000 +0000
+++ tinyerp-client-4.0.0-rc1/bin/tinyerp-client.py	2006-12-05 11:48:16.000000000 +0000
@@ -73,8 +73,8 @@
 import pango
 
 # end testing
-APP = 'terp'
-DIR = 'po'
+APP = 'tinyerp-client'
+DIR = '/usr/share/locale'
 
 locale.setlocale(locale.LC_ALL, '')
 gettext.bindtextdomain(APP, DIR)
